src/cdk-experimental/menu/menu-panel.ts

```diff
@@ -16,5 +16,12 @@ import {CdkMenu} from './menu';
 @Directive({selector: 'ng-template[cdkMenuPanel]', exportAs: 'cdkMenuPanel'})
 export class CdkMenuPanel {
   /** Reference to the child menu component */
-  _menu: CdkMenu;
+  _menu?: Menu;
+
+  constructor(readonly _templateReference: TemplateRef<unknown>) {}
+
+  /** Set the menu component on the menu panel */
```

Comment:
Could you make this method description a little more elaborate? Right now, it doesn't actually convey why this method is important.